.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@ name: SmallRye Build
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Given that you're merging this change into the 3.0.x branch, shouldn't the 3.0.x branch be listed here? 
I'm not sure how that stuff works exactly, but I assume that when pushing into 3.0.x, then the `build.yml` from 3.0.x will be used to decide what jobs to run for this push, so if you don't list 3.0.x there, then it won't run anything.

Or, to make it simpler, just remove the `branches` section and it should run independently on the branch

Reply:
good catch, thanks